src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -10,25 +10,25 @@
 
 public class RegisterServletTest
 {
-  private RegisterServlet registerServlet;
-  private HttpServletRequest mockRequest;
-  private PrintWriter mockPrintWriter;
-  private HttpServletResponse mockResponse;
+    private RegisterServlet registerServlet;
```

Comment:
I think we use two space indents everywhere else in our codebase. You can switch the indent settings on your editor to automatically use 2 spaces instead of four. Please fix.